### PR TITLE
[5.4] Add support for SameSite cookies to CookieJar

### PR DESCRIPTION
**Pull request by iangcarroll**
This relies on Symfony 3.2.2, which includes support for SameSite cookies. https://github.com/symfony/http-foundation/commit/ad11eed7eae097b5097de7c9b5f24d2f2ef416e7

### CODE REVIEW

**Comment by iangcarroll** [comment]
This should be tagged for 5.4, sorry.

File: src/Illuminate/Cookie/CookieJar.php [hunk 1]
```diff
@@ -47,15 +47,22 @@ class CookieJar implements JarContract
...
      */
-    public function make($name, $value, $minutes = 0, $path = null, $domain = null, $secure = false, $httpOnly = true)
+    public function make($name, $value, $minutes = 0, $path = null, $domain = null, $secure = false, $httpOnly = true, $sameSite = null)
     {
         list($path, $domain, $secure) = $this->getPathAndDomain($path, $domain, $secure);
 
         $time = ($minutes == 0) ? 0 : Carbon::now()->getTimestamp() + ($minutes * 60);
 
-        return new Cookie($name, $value, $time, $path, $domain, $secure, $httpOnly);
+        if ($sameSite === 'Lax') {
```
Comment (GrahamCampbell) [inline src/Illuminate/Cookie/CookieJar.php]: why the string comparison?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Cookie/CookieJar.php (hunk 1 above)
Comment (iangcarroll) [inline src/Illuminate/Cookie/CookieJar.php]: Not sure why I thought that was needed.

**Comment by iangcarroll** [comment]
StyleCI seems to be broken.

**Comment by taylorotwell** [comment]
Has a bunch of other changes with it.

**Pull request closed without merging**